src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -496,7 +496,7 @@ protected function style($style, $value, $attr = array())
             $attr['href'] = $href;
         }
         if (isset($attr['href'])) {
-            $v = sprintf('<a href="%s">%s</a>', esc($this->utf8Encode($attr['href'])), $v);
+            $v = sprintf('<a href="%s" target="_blank">%s</a>', esc($this->utf8Encode($attr['href'])), $v);
```

Comment:
Maybe it's good to also add the `rel="noopener noreferrer"` attribute to avoid security issues. See https://medium.com/@jitbit/target-blank-the-most-underestimated-vulnerability-ever-96e328301f4c